core/src/main/java/com/linecorp/armeria/common/logging/ContentPreviewerFactoryCombinable.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.logging;
+
+import com.linecorp.armeria.common.util.Combinable;
+
+@FunctionalInterface
+interface ContentPreviewerFactoryCombinable extends Combinable<ContentPreviewerFactory> {
```

Comment:
This interface (and `Combinable` as well) could be removed if we make `ContentPreviewerFactory` immutable.